src/index.js

```diff
@@ -81,6 +81,12 @@ const getInputfilePath = (targets, shouldThrow = false) => {
     return path.resolve(process.env.PWD, targets[0]);
 }
 
+// Sanity check: make sure that the value was passed to the `output` option
+// Fixes https://github.com/technopagan/sqip/issues/11
+const getOutputFilePath = () => {
+  const index = process.argv.findIndex(arg => arg == '-o' || arg == '--output');
```

Comment:
I would personally prefer using strict equality check `===` and stick with a solution that uses `argv` still.

Reply:
Would you like me try a different approach or only change equality checks? As I understand, we cannot determine if a user specified `true` as a output filename or passed `-o` option without any value if we use [options.output](https://github.com/technopagan/sqip/blob/master/src/index.js#L156) property for it.

Reply:
>  As I understand, we cannot determine if a user specified true as a output filename or passed -o option without any value if we use options.output property for it.

That is true, actually `argv` parses the options correctly since something like `command -o` would be interpreted as `o=true`. But it does not check the value and converts the boolean value to path.

At some point we need to change `argv` with something else that would be the best solution. But for now going with your approach should work. 

@technopagan usually prefers warning/error if something is wrong. If he is also fine with the solution changing equality checks would be enough. 😄 

Cheers,